bitnami/rabbitmq/templates/statefulset.yaml

```diff
@@ -92,8 +92,17 @@ spec:
             - -ec
             - |
               mkdir -p "{{ .Values.persistence.mountPath }}"
+              {{- if eq ( toString ( .Values.volumePermissions.containerSecurityContext.runAsUser )) "auto" }}
+              chown -R `id -u`:`id -G | cut -d " " -f2` {{ .Values.persistence.mountPath }}
```

Comment:
This first command should not be recursive, and only change the ownership of the folder. Equivalent to line 95:

https://github.com/bitnami/charts/blob/74e05c5c3e31b53508cba7365388b6710ba9c854/bitnami/rabbitmq/templates/statefulset.yaml#L95

Reply:
Thanks! updated.